README.md

```diff
@@ -1 +1,72 @@
-# Test_Driven_Development
\ No newline at end of file
+# Test Driven Development Project
+
+This project demonstrates Test Driven Development (TDD) using RSpec for testing. The main objective is to create a class called `Solver` with specific methods, adhering to the given requirements.
+
+## Table of Contents
+  - [Built With](#built-with)
+  - [Tech Stack](#tech-stack)
+  - [Key Features](#key-features)
+  - [Getting Started](#getting-started)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+  - [Usage](#usage)
+  - [Author](#author)
+  - [Future Features](#future-features)
+  - [Contributing](#contributing)
+  - [Show your support](#show-your-support)
+  - [Acknowledgements](#acknowledgements)
+  - [License](#license)
+
+## Built With <a name="built-with"></a>
+- Ruby
+- RSpec
+
+## Tech Stack <a name="tech-stack"></a>
+- Ruby
+- RSpec
+
+## Key Features <a name="key-features"></a>
+- Unit tests using RSpec
+- Class `Solver` with the following methods:
+  - `factorial`: Returns the factorial of a number.
+  - `reverse`: Reverses a given string.
+  - `fizzbuzz`: Implements the FizzBuzz logic.
+
+## Getting Started <a name="getting-started"></a>
+
+### Prerequisites <a name="prerequisites"></a>
+Before you begin, ensure you have [Ruby installed on your machine](https://www.ruby-lang.org/en/downloads/)
+
+### Setup <a name="setup"></a>
+Clone the repository to your local machine.
+  ```bash
+    git clone https://github.com/richie1988/Test_Driven_Development.git
+  ```
+
+### Install <a name="install"></a>
+Run `bundle install` to install the required dependencies.
+
+## Usage <a name="usage"></a>
+Execute the tests using the command `rspec`.
+  ```bash
+    rspec spec spec/solver_spec.rb
+  ```
+## Authors <a name="authors"></a>
+This project was developed by: **[Richard Sikaonga](https://github.com/richie1988)** and **[Nelly Telli](https://github.com/lily4178993)**
+
+## Future Features <a name="future-features"></a>
+No planned future features at this time. The project is complete and functional, but open to community contributions and suggestions for improvements.
+
+## Contributing <a name="contributing"></a>
+Contributions are welcome. Feel free to [open an issue](https://github.com/richie1988/Test_Driven_Development/issues) or create a pull request.
+ or [create a new fork](https://github.com/richie1988/Test_Driven_Development/fork).
+
+## Show your support <a name="support"></a>
+Give a ⭐️ if you found this project interesting.
+
+## Acknowledgements <a name="acknowledgements"></a>
+We would like to acknowledge the contributions and support of the open-source community in making this project possible.
+
+## License <a name="license"></a>
+This project is licensed under the MIT License - see the [LICENSE](./LICENSE.md) file for details.
```

Comment:
- [ ] Well done on the project. However, the link to your license in your readme file is not active. Please correct it. It is critical to include a license in your project because it is a document that provides legally binding guidelines for use and imposes restrictions on how your project can be used. 📗 🙂